build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.CS.targets

```diff
@@ -1,7 +1,23 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <Target Name="GenerateBootstrapCS" 
-            BeforeTargets="BeforeCompile">
+    <PropertyGroup Condition="'$(WindowsAppSDKBootstrapAutoInitialize)'==''">
+        <WindowsAppSDKBootstrapAutoInitialize>true</WindowsAppSDKBootstrapAutoInitialize>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(WindowsAppSDKBootstrapAutoInitialize)'=='true'">
+        <DefineConstants
```

Comment:
this looks ill-formed, and the defines below should be appending, not replacing (this is a property, not an itemgroup)

Reply:
Doh! Only samples I found didn't show any appending but I [just found one](https://stackoverflow.com/questions/36793499/defineconstants-in-addition-to-existing-ones-in-csproj#36793713)

```
<PropertyGroup>
  <DefineConstants>FOO</DefineConstants>
</PropertyGroup>
<PropertyGroup>
  <DefineConstants>$(DefineConstants);BAR</DefineConstants>
</PropertyGroup>
```

That makes sense. Fixing. Thanks